File: 1 ano/Log de Programação/2°Bi/Lista 2/SalaPy/Lista2-sala_Exercício 1 - Média.py
n1 = float(input("Digite sua 1° nota: "))
n2 = float(input("Digite sua 2° nota: "))
n3 = float(input("Digite sua 3° nota: "))
M = (n1 + n2 + n3)/3
if (M >= 6.0):
    print("Você foi aprovado, sua nota foi {:.2f}".format(M))
else:
    print("Você não foi aprovado, sua média foi {:.2f}".format(M))

File: 1 ano/Log de Programação/2°Bi/Lista 2/SalaC/Lista2-sala_Exercício 1 - Média.c
#include <stdio.h>

int main()
{   
    float n1,n2,n3,M;
    
    printf("Digite a 1 nota: ");
    scanf("%f",&n1);
    
    printf("Digite a 2 nota: ");
    scanf("%f",&n2);
    
    printf("Digite a 3 nota: ");
    scanf("%f",&n3);
    
    M = (n1 + n2 + n3) / 3;
    
    if (M >= 6){
        printf("Voce foi aprovado, sua media foi %.2f", M);
    }
    else{
        printf("Voce foi reprovado, sua media foi %.2f", M);
    }
}
